layers/meta-balena-beaglebone/recipes-core/images/balena-image.inc

```diff
@@ -25,6 +27,15 @@ BALENA_BOOT_PARTITION_FILES:beagleboard-xm = " \
     u-boot.img: \
 "
 
+BALENA_BOOT_PARTITION_FILES:beagleplay = " \
+    tispl.bin_unsigned:/tispl.bin \
+    u-boot.img: \
+    tiboot3.bin: \
+    ${KERNEL_IMAGETYPE}${KERNEL_INITRAMFS}-${MACHINE}.bin:/${KERNEL_IMAGETYPE} \
+    k3-am625-beagleplay.dtb: \
+    extlinux.conf:/extlinux/extlinux.conf \
```

Comment:
If only the flasher is using extlinux.conf then this needs to be moved out of here and instead added to balena-image-flasher.bbappend If you add it to the balena-image.inc, it will also be included in the non-flasher image and I believe that one is not supposed to use extlinux.conf

Reply:
I have made this change.